In [9]:
import numpy as np
import pandas as pd

In [10]:
data=pd.read_csv("/content/Q9_a.csv")

In [11]:
data

Index  speed  dist
0       1      4     2
1       2      4    10
2       3      7     4
3       4      7    22
4       5      8    16
5       6      9    10
6       7     10    18
7       8     10    26
8       9     10    34
9      10     11    17
10     11     11    28
11     12     12    14
12     13     12    20
13     14     12    24
14     15     12    28
15     16     13    26
16     17     13    34
17     18     13    34
18     19     13    46
19     20     14    26
20     21     14    36
21     22     14    60
22     23     14    80
23     24     15    20
24     25     15    26
25     26     15    54
26     27     16    32
27     28     16    40
28     29     17    32
29     30     17    40
30     31     17    50
31     32     18    42
32     33     18    56
33     34     18    76
34     35     18    84
35     36     19    36
36     37     19    46
37     38     19    68
38     39     20    32
39     40     20    48
40     41     20    52
41     42     20    56
42     43     20    64
43     44     22    66
44     45     23    54
45     46     24    70
46     47     24    92
47     48     24    93
48     49     24   120
49     50     25    85

In [12]:
# Skewness
data.speed.skew()

-0.11750986144663393

In [13]:
# Skewness
data.dist.skew()

0.8068949601674215

In [14]:
# Skewness
data.skew()

Index    0.000000
speed   -0.117510
dist     0.806895
dtype: float64

## skewness value for speed and distance is -0.117510 and 0.806895 

In [15]:
# Kurtosis
data.speed.kurt()

-0.5089944204057617

In [16]:
# Kurtosis
data.dist.kurt()

0.4050525816795765

In [17]:
# Kurtosis
data.kurt()

Index   -1.200000
speed   -0.508994
dist     0.405053
dtype: float64


## Kurtosis value for speed and distance is -0.508994 and 0.405053
